### ISSUE TITLE
@directus/extension-sdk missing "common" template directory

### ISSUE THREAD

**Issue by tylerforesthauser** [directus/directus#6932]
### Describe the Bug

After updating to 9.0.0-rc.85 and attempting to create a new extension using the `@directus/extension-sdk` package via `npx directus-extension create interface my-custom-interface`, I immediately encounter the following message in my terminal:
```
⠋ Scaffolding Directus extension...(node:28450) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, lstat '/var/www/my-project/node_modules/@directus/extension-sdk/templates/common'
```
I was able to resolve it easily by simply navigating to the `/var/www/my-project/node_modules/@directus/extension-sdk/templates` directory and creating an empty directory called `common`.

### To Reproduce

1. Bootstrap a new Directus project via `npx create-directus-project example-project`.
2. `cd example-project`
3. Install the @directus/extension-sdk package via `npm i -D @directus/extension-sdk`
4. `cd extensions/interfaces`
5. `npx directus-extension create interface my-custom-interface`

### What version of Directus are you using?

9.0.0-rc.85

### What version of Node.js are you using?

v14.15.4

### What database are you using?

Tested with both Postgres 13 and SQLite 3.31.0

### What browser are you using?

n/a

### What operating system are you using?

Debian 4.19.152-1

### How are you deploying Directus?

Locally

**Comment by Attacler** [comment]
I also have this issue, you can solve this by copy and pasting the files manualy because they are in fact there.

**Comment by Attacler** [comment]
also add:

  "directus:extension": {
    "type": "interface",
    "path": "../extensions/interfaces/overzichten/index.js",
    "source": "./overzichten/src/index.js",
    "hidden": false,
    "host": ""
  }
  to your package.json with the correct paths

**Comment by rijkvanzanten** [comment]
Fixed in #7196
